src/jbi/services.py

```diff
@@ -36,17 +47,35 @@ def bugzilla_check_health() -> ServiceHealth:
     return health
 
 
-def jira_check_health() -> ServiceHealth:
+def jira_check_health(actions: Actions) -> ServiceHealth:
     """Check health for Jira Service"""
     jira = get_jira()
     server_info = jira.get_server_info(True)
-    health: ServiceHealth = {"up": server_info is not None}
+    health: ServiceHealth = {
+        "up": server_info is not None,
+    }
+    if server_info is None:
+        return health
+
+    visible_projects = {project["key"] for project in jira_visible_projects(jira)}
+    configured_projects = {
```

Comment:
This _almost_ feels like something that should be added to `Actions`, maybe as a property:
```python
@functools.cached_property
def configured_jira_projects(self) -> Set[str]:
    return {
        action.parameters["jira_project_key"]
        for action in self.__root__
        if "jira_project_key" in action.parameters
    }
```